index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <style>
+      .container{
+        display: flex;
+        flex-direction: row;
+      }
+      .section{
+        display: flex;
+        flex-direction: column;
+      }
+      .navbar{
+        visibility: visible;
+        justify-content: flex-start;
+        flex-direction: column;
+      }
+      @media (max-width: 12450px) { 
+        body{
+          background-color: gray;
+        }
+        
+       }
+    </style>
```

Comment:
It is better to use external styling for your page instead 👌🏾.
It is better to link an external stylesheet in your <head> tag.

Reply:
- [ ] Nice Job so far. Please, note that you need to phrase the comments politely as suggestions instead of order. For example: " please it's better to use external styling for your page" 
- [ ] Please give the student the reason why you asking to use " external styling"